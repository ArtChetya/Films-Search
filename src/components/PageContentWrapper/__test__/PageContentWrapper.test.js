import { shallow } from 'enzyme/build';
import React from 'react';
import { PageContentWrapper } from '..';

describe('PageContentWrapper component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(<PageContentWrapper />);
        expect(component).toMatchSnapshot();
    });
});
