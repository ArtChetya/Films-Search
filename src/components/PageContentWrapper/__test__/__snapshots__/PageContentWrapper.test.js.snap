// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContentWrapper component should be rendered correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "PageContentWrapper-sc-3nvgx4-0",
        "isStatic": true,
        "rules": Array [
          "&&{flex-grow:1;overflow:auto;background-color:rgb(250,250,250);padding:0 5%;}",
        ],
      },
      "displayName": "PageContentWrapper",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "PageContentWrapper-sc-3nvgx4-0",
      "target": "main",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
